packages/vaadin-grid/src/vaadin-grid-event-context-mixin.d.ts

```diff
@@ -27,7 +27,7 @@ interface EventContextMixin<TItem> {
    * This may be the case eg. if the event is fired on the `<vaadin-grid>` element and not any deeper in the DOM, or if
    * the event targets the empty part of the grid body.
    */
-  getEventContext(event: Event): GridEventContext<TItem> | object | null;
+  getEventContext(event: Event): GridEventContext<TItem>;
```

Comment:
Worked on the corresponding Typescript example for cell-focus and this here is super weird to use. `null` is actually never returned. The method can return an empty object, but then it's still easier to use `GridEventContext` instead of `object`, and instead make all props on that type optional.

Reply:
Should also align the [JSDoc in the mixin](https://github.com/vaadin/web-components/blob/b9b07bfc6871e39939987e8ee05b533289f4c2ad/packages/vaadin-grid/src/vaadin-grid-event-context-mixin.js#L31)

Reply:
Done.